### PR TITLE
Added an appdata.xml file

### PR DESCRIPTION
**Pull request by Mailaender**
This adds metadata for Linux software stores. See https://people.freedesktop.org/~hughsient/appdata/ for reference. My main motivation is to get you listed in https://software.opensuse.org/packages/Games

### CODE REVIEW

**Comment by sev-** [comment]
My understanding is that this file then must be installed in every system to /usr/share/appdata/scummvm.appdata.xml. If so, could you then help with modification or at least testing of the Makefile change? The file to modify is ports.mk, line 20 or so.

**Comment by sev-** [comment]
Excellent, thank you! Merging.

**Comment by Mailaender** [comment]
WAIT!

**Comment by sev-** [comment]
ah, wait. Sorry for the late pointing out, but I completely missed that your commit log messages do not conform with our conventions.

Please add 'DISTS:" and "BUILD:" prefixes to the commits, and then I merge.

File: ports.mk [hunk 1]
```diff
@@ -18,6 +18,8 @@ install:
 	$(INSTALL) -c -m 644 $(DIST_FILES_DOCS) "$(DESTDIR)$(docdir)"
 	$(INSTALL) -d "$(DESTDIR)$(datadir)"
 	$(INSTALL) -c -m 644 $(DIST_FILES_THEMES) $(DIST_FILES_ENGINEDATA) "$(DESTDIR)$(datadir)/"
+	$(INSTALL) -d "$(DESTDIR)$(datadir)/applications"
```
Comment (Mailaender) [inline ports.mk]: You have weird definition of `datadir`. It expands to `/usr/share/scummvm/` instead of `usr/share/`.

File: ports.mk [hunk 2]
```diff
@@ -18,6 +18,10 @@ install:
 	$(INSTALL) -c -m 644 $(DIST_FILES_DOCS) "$(DESTDIR)$(docdir)"
 	$(INSTALL) -d "$(DESTDIR)$(datadir)"
 	$(INSTALL) -c -m 644 $(DIST_FILES_THEMES) $(DIST_FILES_ENGINEDATA) "$(DESTDIR)$(datadir)/"
+	$(INSTALL) -d "$(DESTDIR)$(datarootdir)/applications"
```
Comment (sev-) [inline ports.mk]: You also need to put it to the install-strip, and to uninstall targets

**Comment by Mailaender** [comment]
Updated and successfully tested the install rules https://build.opensuse.org/request/show/415957

**Comment by sev-** [comment]
Thanks!

**Pull request merged**